week-1/Homework/extra/2-piping.js

```diff
@@ -16,43 +16,51 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  let sum = a + b;
+  return Number(sum.toFixed(2));
 }
 
-function multiply() {
-
+function multiply(c, d) {
+  let result = c * d;
+  return result;
 }
 
-function format() {
-
+function format(f) {
+  return "£" + f;
 }
 
-const startingValue = 2
+const startingValue = `${format("")}${add(1, 3) + 10 * multiply(1, 2)}`;
```

Comment:
This works, but can you try writing both this, and `calcAll`, without using `+` or `*`? Just using the functions you defined.